README.md

```diff
@@ -46,6 +46,18 @@ To hash a message to *G2*, we currently use the try-and-increment method coupled
 
 We implement fast cofactor multiplication, as the *G2* cofactor is large.
 
+## License
+
+BLS-ZEXE is licensed under either of the following licenses, at your discretion.
+
+Apache License Version 2.0 (LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0)
+MIT license (LICENSE-MIT or http://opensource.org/licenses/MIT)
+Unless you explicitly state otherwise, any contribution submitted for inclusion in BLS-ZEXE by you shall be dual licensed as above (as defined in the Apache v2 License), without any additional terms or conditions.
+
+## Third-Party Software Licenses
```

Comment:
since we are a derived work of ZEXE (am I right?) it's not really a third party license situation, so I think we can skip this second part.